### ISSUE TITLE
render_bundle is serving old (cached?) file after some time

### ISSUE THREAD

**Issue by magicznyleszek** [django-webpack/django-webpack-loader#161]
I have a problem with `render_bundle` serving old JS file after some time. Nothing I try to do helps. I see that the file being served is different than the one in `webpack-stats.json`.

The only thing I am able to do to make it work again is restarting my docker container 🙁

---

This is [my config](https://github.com/kobotoolbox/kpi/blob/master/kobo/settings.py):

```py
WEBPACK_LOADER = {
    'DEFAULT': {
        'BUNDLE_DIR_NAME': 'jsapp/compiled/',
        'POLL_INTERVAL': 0.5,
        'TIMEOUT': 5,
    }
}
```

And I just simply use this in [my html template](https://github.com/kobotoolbox/kpi/blob/master/kpi/templates/index.html):

```
{% load render_bundle from webpack_loader %}
{% render_bundle 'app' 'js' %}
```

**Comment by smutnyleszek** [comment]
I may be onto something! I discovered that our `Dockerfile` is .dockerignoring `webpack-stats.json` and then linking to the file outside. And I think the problem happens when the linking fails, maybe because when file is recreated?

**Comment by detectedstealth** [comment]
I'm finding this same problem when running through docker...

**Comment by ivanfr90** [comment]
Check if you are using any type of `CACHE` configuration in Django's settings.
I was suffering the same problem and I realized that I had setted the MemcachedCache for the local environment. For this reason when any *.js was changed, to see new changes I needed to restart my local environment server.

Otherwise, when you deploy on production, changed files need to be copied to the location where you store STATIC files.
